content/en/docs/KRE/training/_index.md

```diff
@@ -36,3 +40,265 @@ Once deployed the generated KRT file, you can open an entrypoint to your local p
 ### Result example
 
 {{< imgproc kai-training-result-example Resize "1000x" />}}
+
+
+This step-by-step page will guide you creating a new Konstellation version for your project that will help you to
+understand the all the main KRE concepts and allow you testing all the KRE functions.
+
+## Prerequisites
+
+- 
+-
+-
+
+## Step-by-step guide
+
+
+### 1. Create the version structure based on the krt-template
+
+First of all, start creating an empty project with the following folders and files:
+
+
+ ``` bash
+    project
+    └───docs   
+    |   └───README.md # Documentation shown inside KAI Server
+    │
+    └───models    # If the process needs a model to do the inference
+    │   
+    └───src       # Source code of each node
+    │   └───etl
+    │   └───github
+    │   └───dockerhub
+    |
+    └─── metrics
+    │   └───dashboards
+    |
+    |   krt.yml   # krt manifest
+    |   public_input.proto    # proto defining entrypoint Services and Messages.
+    |   internal_nodes.proto  # proto defining Messages that interconnect nodes.
+```
+
+### 2. Fill the `krt.yml`
+
+The KRT file definition can be found in the [KRT](../40_krt/_index.md) page.
```

Comment:
I think we could have it as: "A KRT file definition can be found at..."
By writing "the" it feels like this definition is necessary for the process